chips/nrf52/src/acomp.rs

```diff
@@ -0,0 +1,335 @@
+//! Analog Comparator Peripheral Driver, for nrf52
+//! 
+//! Partially based on sam4l implementation of an analog comparator.
+//!
+//! The comparator (COMP) compares an input voltage (VIN+) against a second input voltage (VIN-). VIN+ can
+//! be derived from an analog input pin (AIN0-AIN7). VIN- can be derived from multiple sources depending on
+//! the operation mode of the comparator.
+//!
+//! Main features of the comparator are:
+//! • Input range from 0 V to VDD
```

Comment:
```suggestion
//! - Input range from 0 V to VDD
```

Reply:
For the docs https://deploy-preview-1551--docs-tockosorg.netlify.com/nrf52/acomp/index.html